buildh/cli.py

```diff
@@ -125,42 +125,83 @@ def parse_cli():
     return options, lipids_info
 
 
-def main():
-    """Main function of buildH.
-
-    Correspond to the entry point `buildH`.
-    """
-    # 1) Parse arguments.
+def entry_point():
+    """Correspond to the entry point `buildH`."""
+    # Parse arguments.
     args, dic_lipid = parse_cli()
 
+    try:
+        main(args.coord, args.traj, args.defop, args.out, args.opdbxtc, args.begin, args.end, dic_lipid)
+    except BuildHError as e:
+        sys.exit(e)
+
+
+def launch(coord_file, def_file, lipid_type, traj_file=None, out_file="OP_buildH.out", prefix_traj_ouput=None, begin=0, end=1, lipid_jsons=None):
```

Comment:
Can you please add docstring?

Reply:
Yes, once we agree about the prototype of the functions and the general implemention, I'll create those.

Reply:
To me the prototype is fine 👍. We have the same arguments as those on the Unix command line, so it should be easy to use!